week1/homework/src/server.js

```diff
@@ -5,11 +5,41 @@ const http = require('http');
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
+
 function createServer(port) {
   let state = 10;
+  function handleRequest(response, fileType) {
+    response.writeHead(200, { 'Content-Type': fileType });
+    response.write(JSON.stringify({ state }));
```

Comment:
I am really happy that you isolated this in a separate function that you are reusing. A small thing that bothers me is that the response body is always` JSON.stringify(state)` i.e. the response content type is always JSON. Thats ok, but if you already know that is the case, does it make sense to have the content-type as a parameter?

Reply:
I thought how I can make it reusable. Yes You are right. In this example always JSON.